app/views/orders/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= simple_form_for @order do |f| %>
+  <%= f.input :user_id, input_html: {value: current_user.id} %>
+  <%= f.fields_for :order_meals do |o_m_f| %>
+    <%= o_m_f.input :meal_id, collection: Menu.last.meals, label_method: lambda {|meal| "#{meal.food_item.name}"} %>
```

Comment:
this lambda has to be a method in decorator

use `->` shortcut instead of `lambda`
[source](https://github.com/drapergem/draper)